### PR TITLE
Property uses `generics_def`

### PR DESCRIPTION
**Pull request by PerisicJere**

### CODE REVIEW

File: library/tests/fy_py_files/test_property_wtih_generics.py
Comment (PerisicJere) [inline library/tests/fy_py_files/test_property_wtih_generics.py:1]: test

File: library/tests/fy_py_files/test_fy_py_files/property_with_generics/property_with_generics_fy.py.expected
Comment (PerisicJere) [inline library/tests/fy_py_files/test_fy_py_files/property_with_generics/property_with_generics_fy.py.expected:1]: expected test

File: library/tests/fy_py_files/test_fy_py_files/property_with_generics/property_with_generics_fy.py
Comment (PerisicJere) [inline library/tests/fy_py_files/test_fy_py_files/property_with_generics/property_with_generics_fy.py:1]: actual test

File: library/tests/fy_py_files/test_fy_py_files/property_with_generics/greetings_t.py
Comment (PerisicJere) [inline library/tests/fy_py_files/test_fy_py_files/property_with_generics/greetings_t.py:1]: generics for test

File: library/src/fy_library/mixins/property/generated_fy_py_code/jinja2_templates/property.jinja2 [hunk 1]
```diff
@@ -5,8 +5,16 @@ class {{ python_class_name.pascal_case }}
 {%- for abstract_property in abstract_property_mixins %}
     {{ abstract_property.python_class_name.pascal_case }},
 {%- endfor %}
+    {%- if generics_def %}
+    Generic[{{ generics_def }}],
+    {%- endif %}
     abc.ABC,
 )
+{%- endif %}
```
Comment (vinkobuble) [inline library/src/fy_library/mixins/property/generated_fy_py_code/jinja2_templates/property.jinja2:13]:
```suggestion
{%- else %}
```

File: library/src/fy_library/mixins/property/generated_fy_py_code/jinja2_templates/property.jinja2 [hunk 2]
```diff
@@ -5,8 +5,16 @@ class {{ python_class_name.pascal_case }}
...
+    {%- if generics_def and not abstract_property_mixins -%}
```
Comment (vinkobuble) [inline library/src/fy_library/mixins/property/generated_fy_py_code/jinja2_templates/property.jinja2:14]:
```suggestion
    {%- if generics_def -%}
```

**Review by vinkobuble — APPROVED** [review]

**Pull request merged**